R/Rcpp.package.skeleton.R

```diff
@@ -218,6 +213,11 @@ Rcpp.package.skeleton <- function(name = "anRpackage", list = character(),
         rm("rcpp_hello_world", envir = env)
     }
 
+    if (attributes) {
+        compileAttributes(root)
+        message(" >> compiled Rcpp attributes ")
+    }
+
```

Comment:
Maybe I am short of coffee bu `if (attributes) {` relies on `attributes==TRUE`.  We could have a case of `modules=TRUE` (as in #704) _and also_ `attributes=FALSE`, no?   Needless complication?